services/app-api/src/authn/cognitoAuthn.ts

```diff
@@ -126,45 +144,153 @@ export function userTypeFromAttributes(attributes: {
             role: 'CMS_USER',
             email: attributes.email,
             name: fullName,
+            givenName: attributes.given_name,
+            familyName: attributes.family_name,
+            euaID: euaID,
```

Comment:
I think these two functions are handling the admin user differently. We have code on the prisma side that handles the admin but it looks like the cognito side doesn't have this and admin would fall in the error "unsupported user role" block.

Reply:
fyi I made a ticket where this could be handled (adding full admin user workflow) https://github.com/CMSgov/managed-care-review/pull/1329